app/models/user.rb

```diff
@@ -214,5 +222,32 @@ def assign_default_role
   def email_messages_enabled?
     email_messages_enabled
   end
+
+  def collections_with_messaging_dis
```

Comment:
I'm not sure what `dis` means in this context. I'm also not sure that this method is used? It might be implicit.